test/unit/spec/OktaSignIn_spec.js

```diff
@@ -89,5 +96,35 @@ function (Widget, Expect, Logger) {
         expect(signIn.token.parseTokensFromUrl).toBeDefined();
       });
     });
+
+    Expect.describe('getTransaction', function () {
+      it('resolves to an error if a transaction cannot resume', function (done) {
+        signIn.getTransaction()
+        .then(function(){/* Should never reach this */})
+        .catch(function(err) {
+          expect(err.name).toBe('AuthSdkError');
+          expect(err.errorSummary).toBe('No transaction to resume');
+          done();
+        });
+      });
+      it('calls the authentication api with a stateToken', function (done) {
```

Comment:
It may be simpler to mock `authJs.resume` if possible, and just assert that what we pass to `getTransaction()` gets passed to the `authJS.resume()` transaction mock.  Similarly, with a `authJs.resume` mock, you could mock the return value and assert it makes it's way back through `getTransaction()`.